spec/chewy/index/import_spec.rb

```diff
@@ -270,8 +270,8 @@ def subscribe_notification
         expect(payload).to eq(
           errors: {
             index: {{
-              'type' => 'mapper_parsing_exception',
-              'reason' => 'object mapping for [object] tried to parse field [object] as object, but found a concrete value'
+              'type' => 'document_parsing_exception',
+              'reason' => '[1:27] object mapping for [object] tried to parse field [object] as object, but found a concrete value'
```

Comment:
Are these numeric prefixes (`[1:27]` in this example) new error-code constants from the ES client gem ... or contingent on the spec data setup here, or something else?

Reply:
Don't know this yet, I've just made all specs passing, to see how many things do we break by upgrading to ES 8.x

Reply:
When can we expect for this to be considered for release?